### PR TITLE
[WIP] Add seed selection screen to mobile.

### LINKED ISSUE #579: Implement Seed / Recovery select screen mobile
https://invis.io/5TNS92K6N3V
https://invis.io/K7NS94FE5WZ

### PR DESCRIPTION
**Pull request by valentinewallace**
Selection screen is done, but I'm still working on the actual seed screens. Still only testing on Android so far.

Closes #579

### CODE REVIEW

File: src/view/select-seed.js [hunk 1]
```diff
@@ -5,53 +5,57 @@ import { observer } from 'mobx-react';
...
-import { color } from '../component/style';
+import { color, breakWidth } from '../component/style';
 
 //
 // Select Seed View
 //
 
 const baseStyles = {
   content: {
-    justifyContent: 'center',
```
[the remark this answers is not in the document]
Comment (valentinewallace) [reply src/view/select-seed.js]: @tanx That's my only question. Otherwise, I'm good to merge!

[the remark this answers is not in the document]
Comment (tanx) [reply src/view/select-seed.js]: You guys keep resizing the window to these totally unrealistic sizes 😂

[the remark this answers is not in the document]
Comment (tanx) [reply src/view/select-seed.js]: This is Lee's design...

bildschirmfoto 2018-11-13 um 10 56 44

[the remark this answers is not in the document]
Comment (tanx) [reply src/view/select-seed.js]: > Otherwise, I'm good to merge!

ok thanks 🙏

**Pull request merged**